### DISCUSSION TITLE
Any way to apply custom branding? (self-host)

### DISCUSSION THREAD

**Discussion by userx8zWEThCXy633Tqrb** [stoatchat/discussions#327] [Q&A] [answered]
Hello, I am self hosting an instance of Revolt. I was wondering if there is any easy way to customize the branding? For example, change the "CHAT" svg to a logo of our own?
Thanks

**Comment by Zomatree** [comment, marked as answer]
You use the [setup_assets.js](https://github.com/revoltchat/revite/blob/master/scripts/setup_assets.js) script which can be accessed through `yarn pull`.
